README.md

```diff
@@ -17,6 +17,33 @@ npm install react-native-checkbox-reanimated
 npm install react-native-reanimated react-native-svg
 ```
 
+When you're done installing ```react-native-reanimated``` don't forget to add the plugin to your ```babel.config.js``` file. Like so:
```

Comment:
Too many backticks for inline codes. It should be

    `react-native-reanimated`

https://www.markdownguide.org/basic-syntax/#code

Reply:
I've made the corrections